config/locales/en.yml

```diff
@@ -1122,8 +1122,8 @@ en:
     uncomplete_module: "<i>%{user}</i> uncompleted task <strong>%{module}</strong>."
     assign_sample: "<i>%{user}</i> assigned sample(s) <strong>%{samples}</strong> to task(s) <strong>%{tasks}</strong>"
     unassign_sample: "<i>%{user}</i> unassigned sample(s) <strong>%{samples}</strong> from task(s) <strong>%{tasks}</strong>"
-    assign_repository_records: "<i>%{user}</i> assigned <strong>%{repository}</strong> repository records(s) <strong>%{records}</strong> to task <strong>%{task}</strong>"
-    unassign_repository_records: "<i>%{user}</i> unassigned <strong>%{repository}</strong> repository records(s) <strong>%{records}</strong> from task <strong>%{task}</strong>"
+    assign_repository_records: "<i>%{user}</i> assigned records(s) <strong>%{records}</strong> from <strong>%{repository}</strong> repository to task <strong>%{task}</strong>"
+    unassign_repository_records: "<i>%{user}</i> unassigned records(s) <strong>%{records}</strong> from <strong>%{repository}</strong> repository from task <strong>%{task}</strong>"
```

Comment:
I agree, but I was discussing this wording with Hana and she said that it's good enough for now, because they are planing in the near future to change the wording throughout the whole application.